mk/linux/setupBuildDeps.sh

```diff
@@ -145,7 +145,7 @@ case $distribution in
 				;;
 			14.04*)
 				#LTS, name > trusty, EoL April 2019
-				installcommand="apt-get install $APT_OPTIONS build-essential cmake libsdl2-dev libalut-dev libgl1-mesa-dev libglu1-mesa-dev libvorbis-dev libwxgtk3.0-dev libx11-dev liblua5.1-0-dev libjpeg-dev libpng12-dev libcurl4-gnutls-dev libxml2-dev libircclient-dev libglew-dev libftgl-dev libfribidi-dev libvlc-dev libvlccore-dev libcppunit-dev"
+				installcommand="apt-get install $APT_OPTIONS build-essential cmake libalut-dev libgl1-mesa-dev libglu1-mesa-dev libvorbis-dev libwxgtk3.0-dev libx11-dev liblua5.1-0-dev libjpeg-dev libpng12-dev libcurl4-gnutls-dev libxml2-dev libircclient-dev libglew-dev libftgl-dev libfribidi-dev libvlc-dev libvlccore-dev libcppunit-dev"
```

Comment:
Are we still supporting trusty? I had to remove the libsdl2-dev dep to get the travis build to work. the dpendency won't install on trusty due to conflicts, so the source tarball is downloaded and built instead.

If we're still supporting trusty, we may need to add that to this script the way it's done in .travis.yml in the `before_install` section.

Reply:
I think we only ever meant to support OS releases during their lifespans.

Travis CI build nodes used to be running old, and default to outdated, Ubuntu releases, though. Fortunately this [does not seem to be the case now](https://docs.travis-ci.com/user/reference/overview/#virtualization-environments) (defaults to Ubuntu 16.04 LTS "Xenial", Ubuntu 18.04 LTS "Bionic" is available).